src/quarto.py

```diff
@@ -0,0 +1,33 @@
+def main():
+    print("Welcome to Quarto-Py")
+    startGame()
+
+
+def startGame():
+    grid = initGrid()
+    displayGrid(grid)
+    print("Game Started")
+
+
+def initGrid(gridSize=4):
+    grid = []
+    i = 0
+    while i < gridSize:
+        grid.append([])
+        j = 0
+        while j < gridSize:
+            grid[i].append('.')
+            j += 1
+        i += 1
+    return grid
+
+
+def displayGrid(grid):
+    for line in grid:
```

Comment:
you should separate the function that transforms the grid into a string, from the function that prints that string. The first function would be easy to test.